src/openlcb/MemoryConfig.hxx

```diff
@@ -787,11 +787,11 @@ private:
             available_commands |= MemoryConfigDefs::AVAIL_SR;
         }
         // Figure out about ACDI spaces
-        MemorySpace* memspace = registry_.lookup(message()->data()->dst, 0xFC);
+        MemorySpace* memspace = get_space(0xFC);
```

Comment:
Should we consider removing the magic numbers (0xFC, 0xFB, etc) and use the contants defind in MemoryConfigDiffs.hxx instead?

Reply:
This is generally a good idea, but not in this particular place, since the code in the next line has
`MemoryConfigDefs::AVAIL_R0xFC`.